### ISSUE TITLE
False positive for too-many-function-args with numpy

### ISSUE THREAD

**Issue by RJB17** [PyCQA/pylint#3059]
Reoccurence of [2326](https://github.com/PyCQA/pylint/issues/2326)
### Steps to reproduce
1. 
```
astroid==2.2.5
isort==4.3.21
lazy-object-proxy==1.4.1
mccabe==0.6.1
numpy==1.17.0
pylint==2.3.1
six==1.12.0
typed-ast==1.4.0
wrapt==1.11.2
python==3.7
```
2.
```
import numpy as np
X = np.timedelta64(1, 'D')
```
3. Run pylint 

### Current behavior
E1121: Too many positional arguments for constructor call (too-many-function-args)

### Expected behavior
No too-many-function-args
### pylint --version output
pylint 2.3.1
astroid 2.2.5
Python 3.7.2 (default, Jun  7 2019, 15:59:38)
[GCC 7.4.0]

**Comment by PCManticore** [comment]
Thanks @RJB17 !

**Comment by hippo91** [comment]
@RJB17 
I can confirm your problem with `pylint 2.3.1` and `astroid 2.2.5`. However with the current master branch of `astroid` the problem disappears.
The emission of such message will thus disappear with the next version of  `astroid`.

**Comment by PCManticore** [comment]
Thanks for confirming that this is already fixed @hippo91 Let's close this for now.
